cvmfs/cvmfs_server

```diff
@@ -4046,8 +4052,20 @@ snapshot() {
       -i ${spool_dir}/tmp/last_snapshot \
       -o .cvmfs_last_snapshot"
 
-    # remove magic file after successful replication
-    $user_shell "$delete_command"
+    # run the automatic garbage collection (if configured)
+    if is_garbage_collectable $name && [ ! -z "$CVMFS_GC_TIMESTAMP_THRESHOLD" ]
```

Comment:
The `Gyes` in *.cvmfspublished* indicates that a repository is garbage collectible to the outside world (for example stratum1s). On the stratum 0 the garbage collectibility is determined by `CVMFS_GARBAGE_COLLECTION=true` and the flag in *.cvmfspublished* is set accordingly.

I agree that stratum 1s should be able to determine the garbage collectibility solely based on the `Gyes` in *.cvmfspublished* omitting the `CVMFS_GARBAGE_COLLECTION=true` all together and make this a stratum0-only configuration variable. I'll look into that.